test/unittests/serviceRoutines/statisticsTreat_test.cpp

```diff
@@ -64,10 +65,9 @@ TEST(statisticsTreat, delete)
   out       = restService(&ci, rs);
   EXPECT_STREQ(expectedBuf, out.c_str());
 
-  EXPECT_EQ("OK", testDataFromFile(expectedBuf, sizeof(expectedBuf), outfile2)) << "Error getting test data from '" << outfile2 << "'";
   ci.outFormat = JSON;
   out       = restService(&ci, rs);
-  EXPECT_STREQ(expectedBuf, out.c_str());
+  EXPECT_STREQ("{\"message\":\"All statistics counter reset\"}", out.c_str());
```

Comment:
I imagine you tried to simply *remove* the last newline from the file?
I have a feeling your editor adds it again without asking you when it saves the file ... Seen it before (not with emacs, of course ;-)).